app/src/main/res/layout/activity_main.xml

```diff
@@ -18,86 +18,89 @@
+                android:lineSpacingExtra="0sp"
+                />
 
-        <com.google.android.material.floatingactionbutton.FloatingActionButton
-            android:id="@+id/actionButton"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_gravity="bottom|end"
-            android:layout_margin="@dimen/dp_16"
-            android:scaleType="center"
-            android:src="@drawable/ic_add_white_24dp"
-            app:backgroundTint="@color/blue"
-            app:borderWidth="0dp"
-            app:elevation="3dp"
-            app:fabSize="normal"
-            app:layout_behavior=".features.main.FabBehavior"/>
+            <ImageView
+                android:id="@+id/appBarActionQrCode"
+                android:layout_width="@dimen/dp_24"
+                android:layout_height="@dimen/dp_24"
+                android:layout_marginRight="@dimen/dp_24"
+                android:src="@drawable/ic_action_qr_code"
+                />
 
-        <View
-            android:id="@+id/bottomDivider"
-            android:layout_width="match_parent"
-            android:layout_height="0.1dp"
-            android:layout_gravity="bottom" />
-    </androidx.coordinatorlayout.widget.CoordinatorLayout>
+            <ImageView
+                android:id="@+id/appBarActionMenu"
+                android:layout_width="@dimen/dp_24"
+                android:layout_height="@dimen/dp_24"
+                android:layout_marginRight="@dimen/dp_16"
+                android:src="@drawable/ic_action_menu"
+                />
+        </LinearLayout>
+    </androidx.constraintlayout.widget.ConstraintLayout>
 
-    <LinearLayout
-        android:id="@+id/bottomNavigationLayout"
+    <FrameLayout
+        android:id="@+id/container"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_alignParentBottom="true"
-        android:orientation="vertical">
-
-        <com.google.android.material.bottomnavigation.BottomNavigationView
-            android:id="@+id/bottomNavigationView"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            app:itemBackground="@android:color/white"
-            app:itemIconTint="@drawable/selector_bottom_bar_item"
-            app:itemTextColor="@color/blue"
-            app:labelVisibilityMode="unlabeled"
-            app:menu="@menu/main_navigation" />
-    </LinearLayout>
+        android:layout_height="0dp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/appBarLayout" />
 
     <com.saltedge.authenticator.features.security.UnlockAppInputView
         android:id="@+id/unlockAppInputView"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:layout_alignParentTop="true" />
-</RelativeLayout>
+        android:visibility="gone"
```

Comment:
Also u can use here smth like this:
`android:visibility="@{viewModel.layoutVisibility}"`

Reply:
i will add binding in the next PR